### ISSUE TITLE
eliminate the suggested pbcore vocabulary for pbcoreGenre

### ISSUE THREAD

**Issue by johnnypass** [WGBH/PBCore2.0#70]
Can we eliminate the suggested pbcore vocabulary for pbcoreGenre? I'm not sure about the origins of the list, but I'm not finding terms in there that don't exist in the other five well-maintained vocabs.  Are there specific terms in the pbcore genre list that don't appear elsewhere?   

http://metadataregistry.org/concept/list/page/1/vocabulary_id/148.html

**Comment by kvanmalssen** [comment]
I agree that this needs to be reviewed, especially in relation to other vocabularies, such as Asset Type.

**Comment by thompease** [comment]
And also pbcoreSubject

**Comment by marymiller61** [comment]
I think it's better to rely on existing vocabularies than to try to cobble together our own.

**Comment by jolene2323** [comment]
Agree!
